docs/source/reference/io.rst

```diff
@@ -59,6 +59,18 @@ Node-Link JSON
 .. autofunction:: pybel.from_nodelink_file
 .. autofunction:: pybel.to_nodelink_file
 
+Alternative
+-----------
+As an alternative to previous JSON export functions, PyBEL also provides a customized JSON exporter prior node
```

Comment:
All of this documentation should be in the module docstring of `pybel.io.custom_json`